src/cheri_insts.sail

```diff
@@ -89,20 +126,33 @@ function clause execute (CGetLen(rd, cs1)) =
   RETIRE_SUCCESS
 }
 
+/*!
+ * The low bit of integer register *rd* is set to the **tag** value of *cs1*.
```

Comment:
"tag value of cs1" reads strangely; use field instead like the others?

Reply:
Sounds good, will update the MIPS description too.